crates/sui-sdk/examples/governance_api.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+mod utils;
+
+use sui_sdk::SuiClientBuilder;
+use utils::sui_address_for_examples;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let sui_local = SuiClientBuilder::default().build_localnet().await?; // local Sui network
+    println!("Sui local version{:?}", sui_local.api_version());
+    // create a random Sui address for examples. Check utils module if you want to use a local wallet, or use SuiAddress::from_str("sui_address") for a specific address
+    let active_address = sui_address_for_examples().await?;
+
+    // ************ GOVERNANCE API ************ //
+
+    // Stakes
+    let stakes = sui_local
+        .governance_api()
+        .get_stakes(active_address)
+        .await?;
+
+    println!(" *** Stakes ***");
+    println!("{:?}", stakes);
+    println!(" *** Stakes ***\n");
+
+    // Committee Info
+    let committee = sui_local.governance_api().get_committee_info(None).await?; // None defaults to the last epoch
+
+    println!(" *** Committee Info ***");
+    println!("{:?}", committee);
```

Comment:
Building on my comment on `coin_api.rs` -- rather than just printing the output, how about, e.g. combining these APIs so that you can display the validator details for the coins you've staked, or maybe you calculate your portion of that validator's stake, in basis points?

Reply:
Great idea, will do it!